src/modules/75-cf/components/CreateFlagButton/CreateFlagButton.tsx

```diff
@@ -44,13 +45,14 @@ const CreateFlagButton = (props: CreateFlagButtonProps): ReactElement => {
       text={getString('cf.featureFlags.newFlag')}
       icon="plus"
       intent="primary"
-      variation={ButtonVariation.PRIMARY}
+      variation={isLinkVariation ? ButtonVariation.LINK : ButtonVariation.PRIMARY}
       onClick={() => {
         trackEvent(FeatureActions.AddNewFeatureFlag, {
           category: Category.FEATUREFLAG
         })
         showModal()
       }}
+      margin={isLinkVariation ? { top: 'xlarge' } : {}}
```

Comment:
can this be shortened to: 
```suggestion
      margin={isLinkVariation && { top: 'xlarge' }}
```

Reply:
This is v nice ^ ty